pkgs/applications/window-managers/hyprwm/xdg-desktop-portal-hyprland/default.nix

```diff
@@ -2,74 +2,56 @@
   lib,
   stdenv,
   fetchFromGitHub,
-  fetchpatch,
   cmake,
-  pkg-config,
-  wayland-scanner,
   makeWrapper,
+  pkg-config,
   wrapQtAppsHook,
   nix-update-script,
+  hyprland,
   hyprland-protocols,
   hyprlang,
+  hyprutils,
+  hyprwayland-scanner,
   libdrm,
   mesa,
   pipewire,
   qtbase,
   qttools,
   qtwayland,
   sdbus-cpp,
+  slurp,
```

Comment:
Why add slurp if it's not used anywhere?

Reply:
it was there before, but haven't tested it whether it is actually needed. It's `binPath` is added to `hyprland-share-picker` wrapper.

Reply:
Right, my bad then.